Sources/View/Widget/ExpiryDatePickerView.swift

```diff
@@ -88,7 +88,9 @@ final class ExpiryDatePickerView: UIPickerView, UIPickerViewDelegate, UIPickerVi
     }
 
     private func setupMonths() {
-        months = DateFormatter().monthSymbols.map({ $0.capitalized })
+        let dateFormatter = DateFormatter()
+        dateFormatter.locale = Locale(identifier: Locale.preferredLanguages[0])
+        months = dateFormatter.standaloneMonthSymbols.map({ $0.capitalized })
```

Comment:
We could we something like this to localize the year as well (this is just an example):
```
//
// Copyright 2018 - Present Hyperwallet
//
// Permission is hereby granted, free of charge, to any person obtaining a copy of this software
// and associated documentation files (the "Software"), to deal in the Software without restriction,
// including without limitation the rights to use, copy, modify, merge, publish, distribute,
// sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
// furnished to do so, subject to the following conditions:
//
// The above copyright notice and this permission notice shall be included in all copies or
// substantial portions of the Software.
//
// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.

import UIKit

final class ExpiryDatePickerView: UIPickerView, UIPickerViewDelegate, UIPickerViewDataSource {
    var month: Int!
    var year: Int!

    private var months = [String]()
    private var years = [Int]()
    private var yearsString = [String]()

    convenience init(value: String?, frame: CGRect = .zero) {
        self.init(frame: frame)

        if let values = value?.components(separatedBy: "-"),
            values.count == 2,
            let defaultMonth = Int(values[1]),
            let defaultYear = Int(values[0]) {
            month = defaultMonth
            year = defaultYear
        } else {
            let currentDate = Date()
            let currentCalendar = Calendar.current
            month = currentCalendar.component(.month, from: currentDate)
            year = currentCalendar.component(.year, from: currentDate)
        }

        setupMonths()
        setupYears(add: 10)

        delegate = self
        dataSource = self

        // pick current month as default month for the picker and place the selected month in the center of picker
        selectRow(month - 1, inComponent: 0, animated: false)
    }

    // MARK: UIPicker Delegate / Data Source
    func numberOfComponents(in pickerView: UIPickerView) -> Int {
        return 2
    }

    func pickerView(_ pickerView: UIPickerView, didSelectRow row: Int, inComponent component: Int) {
        month = selectedRow(inComponent: 0) + 1
        year = years[selectedRow(inComponent: 1)]
    }

    func pickerView(_ pickerView: UIPickerView, numberOfRowsInComponent component: Int) -> Int {
        switch component {
        case 0:
            return months.count

        case 1:
            return years.count

        default:
            return 0
        }
    }

    func pickerView(_ pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String? {
        switch component {
        case 0:
            return months[row % months.count]

        case 1:
            return yearsString[row]//String(describing: years[row])

        default:
            return nil
        }
    }

    private func setupMonths() {
        let dateFormatter = DateFormatter()
        dateFormatter.locale = Locale(identifier: Locale.preferredLanguages[0])
        months = dateFormatter.standaloneMonthSymbols.map({ $0.capitalized })
    }

    private func setupYears(add: Int) {
        years = Array(year...year + add)
        let dateFormatter = DateFormatter()
        dateFormatter.dateFormat = "yyyy"

        yearsString = years.map { yearDateFormatter.string(from: dateFormatter.date(from: String($0))!) }
    }

    lazy var yearDateFormatter: DateFormatter = {
        let formatter = DateFormatter()
        formatter.locale = Locale(identifier: Locale.preferredLanguages[0])
        formatter.setLocalizedDateFormatFromTemplate("y")
        return formatter
    }()
}
```

Reply:
The picker will look like this:
![Screen Shot 2019-06-19 at 4 43 29 PM](https://user-images.githubusercontent.com/28455330/59808556-9c42c900-92b1-11e9-9f8a-6e5fd197cd56.png)

Reply:
implemented with some improvements